File: Python/BreakingRecords.py
def breakingRecords(scores):
    # Write your code here
    breakMin = 0
    minScore = 0
    maxScore = 0
    breakMax = 0
    
    minScore = scores[0]
    maxScore = scores[0]
    for score in scores[1:]:
        if score < minScore:
            breakMin = breakMin + 1
            minScore = score
        elif score > maxScore:
            breakMax = breakMax + 1
            maxScore = score
    return [breakMax, breakMin]

File: C#/BreakingRecords.cs
public static List<int> breakingRecords(List<int> scores)
{
    int minBreak = 0;
    int maxBreak = 0;
    
    int minScore = scores[0];
    int maxScore = scores[0];
    
    List<int> res = new List<int>();
    
    foreach(int score in scores){
        
        if(score < minScore){
            minBreak = minBreak + 1;
            minScore = score;
        }else if(score > maxScore){
            maxBreak = maxBreak + 1;
            maxScore = score;
        }
    }
    res.Add(maxBreak);
    res.Add(minBreak);
    
    return res;
}